client/my-sites/checkout/composite-checkout/payment-method-helpers.js

```diff
@@ -148,8 +148,9 @@ export function WordPressCreditsLabel( { credits } ) {
 	return (
 		<React.Fragment>
 			<div>
-				{ translate( 'WordPress.com Credits: %(amount)s', {
+				{ translate( 'WordPress.com Credits: %(amount)s available', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.